src/pages/DataSharingLanguageTool.js

```diff
@@ -0,0 +1,299 @@
+              onChange: (e) => setNmds(e.target.checked),
+              id: 'checkNMDS',
+              type: 'checkbox',
+            }),
+            label({
+              style: labelStyle,
+              className: 'regular-checkbox',
+              htmlFor: 'checkNMDS',
+            }, ['No methods development or validation studies (NMDS)']),
+          ]),
+
+          div({className: 'checkbox'}, [
+            input({
+              checked: gso,
+              onChange: (e) => setGso(e.target.checked),
+              id: 'checkGSO',
+              type: 'checkbox',
+            }),
+            label({
+              style: labelStyle,
+              className: 'regular-checkbox',
+              htmlFor: 'checkGSO',
+            }, ['Genetic Studies Only (GSO)']),
+          ]),
+
+          div({className: 'checkbox'}, [
+            input({
+              checked: pub,
+              onChange: (e) => setPub(e.target.checked),
+              id: 'checkPUB',
+              type: 'checkbox',
+            }),
+            label({
+              style: labelStyle,
+              className: 'regular-checkbox',
+              htmlFor: 'checkPUB',
+            }, ['Publication Required (PUB)']),
+          ]),
+
+          div({className: 'checkbox'}, [
+            input({
+              checked: col,
+              onChange: (e) => setCol(e.target.checked),
+              id: 'checkCOL',
+              type: 'checkbox',
+            }),
+            label({
+              style: labelStyle,
+              className: 'regular-checkbox',
+              htmlFor: 'checkCOL',
+            }, ['Collaboration Required (COL)']),
+          ]),
+
+          div({className: 'checkbox'}, [
+            input({
+              checked: irb,
+              onChange: (e) => setIrb(e.target.checked),
+              id: 'checkIRB',
+              type: 'checkbox',
+            }),
+            label({
+              style: labelStyle,
+              className: 'regular-checkbox',
+              htmlFor: 'checkIRB',
+            }, ['Ethics Approval Required (IRB)']),
+          ]),
+
+          div({className: 'checkbox'}, [
+            input({
+              checked: gs,
+              onChange: (e) => setGs(e.target.checked),
+              id: 'checkGS',
+              type: 'checkbox',
+            }),
+            label({
+              style: labelStyle,
+              className: 'regular-checkbox',
+              htmlFor: 'checkGS',
+            }, ['Geographic Restriction (GS-)']),
+          ]),
+
+          div({className: 'checkbox'}, [
+            input({
+              checked: mor,
+              onChange: (e) => setMor(e.target.checked),
+              id: 'checkMoratorium',
+              type: 'checkbox',
+            }),
+            label({
+              style: labelStyle,
+              className: 'regular-checkbox',
+              htmlFor: 'checkMoratorium',
+            }, ['Publication Moratorium (MOR)']),
+          ]),
```

Comment:
this checkbox is pointless currently because the translateDataRestriction method will never generate any language for this property, so we might as well remove it. Additionally, translateDataRestriction checks for and generates language for POA property, POA is intentionally not included in the DSLT but this may cause some discrepancies between the language generated here and the language generated in actual practice

Reply:
We have not decided on any language for this particular dataset restriction, so yes, there is currently no effect to selecting it. I think the right course of action should be to remove it for this PR, and file a new ticket to add it back in alongside the required language for this option. Let's follow up with @solideoglori for his 👍🏽